README.md

```diff
@@ -74,7 +60,25 @@ Mastodon acts as an OAuth2 provider, so 3rd party apps can use the REST and Stre
 
 The repository includes deployment configurations for **Docker and docker-compose** as well as specific platforms like **Heroku**, and **Scalingo**. For Helm charts, reference the [mastodon/chart repository](https://github.com/mastodon/chart). The [**standalone** installation guide](https://docs.joinmastodon.org/admin/install/) is available in the documentation.
 
-## Development
+### Codebase structure
```

Comment:
I'm not sure this entire section is useful. Some of the descriptions here are either wrong, or sort of correct but phrased a little oddly.

Reply:
Better?

```diff
  ├──.devcontainer                     // Scripts and preparation files for running Mastodon in a development container such as github.dev
  ├──.github                           // Health files for GitHub such as issue templates
- ├──app                               // Collection of code files relating to the app's frontend
+ ├──app                               // The app frontend
  ├──bin                               // Scripts for running and hosting Mastodon
  ├──config                            // Code files relating to federated hosting and Ruby bootstrappers
  ├──db                                // Database files for Mastodon's information database (Host-specific backend)
- ├──dist                              // Collection of code files relating app hosting on domains and port relations
+ ├──dist                              // systemd services for running and hosting Mastodon and using nginx
- ├──lib                               // Code and file library
+ ├──lib                               // Library for functions used across the codebase
  ├──log                               // Hosting logs
  ├──public                            // Static resources
- ├──spec                              // Helpers, presenters, and other code models
+ ├──spec                              // Helpers, presenters, services, and other backend models
  ├──streaming                         // Streaming API service
  └──vendor
```

Reply:
High level - I'd probably delete this or add it to a developer-specific doc file, not the top level README.

More specifically:

- `app` has basically all the files (frontend, backend, activitypub, rest api, the core web app, db models, etc) - not just the frontend
- `bin` has mostly developer-useful tools, and only sort of hosting scripts
- `config` has a grab bag of stuff, not just federated hosting
- `spec` has the test suite
- Some of the others -- .github, log, public, streaming, db -- your descriptions are fine, but also sort of superfluous because the directory is pretty self explanatory and hopefully not confusing to anyone who needs to understand it?

Reply:
Perhaps better to just remove it entirely.

Reply:
I think it probably makes sense to remove this section. Rails developers would likely be familiar with the basic structure.

Reply:
Done! Being a .NET developer probably doesn't help with something like this 😁